packages/common/src/ui/components/popover/HoverPopover/HoverPopover.tsx

```diff
@@ -14,14 +14,16 @@ export const HoverPopover: React.FC<PopoverProps> = ({
   const { show, hide, Popover } = usePopover();
 
   return (
-    <div
-      style={{ cursor: 'help' }}
-      onMouseOver={show}
-      onMouseLeave={hide}
-      onClick={show}
-    >
+    <>
+      <div
+        style={{ cursor: 'help' }}
+        onMouseOver={show}
+        onMouseLeave={hide}
+        onClick={show}
+      >
+        {children}
+      </div>
```

Comment:
offroad - was annoying me to hover over the status thing and the hover popover follows your mouse when you move it in certain directions!